release-notes/10.0/preview/preview2/runtime.md

```diff
@@ -8,6 +8,167 @@
 
 - [What's new in .NET 10](https://learn.microsoft.com/dotnet/core/whats-new/dotnet-10/overview) documentation
 
-## Feature
+## Array Enumeration De-Abstraction
 
-This is about the feature
+Preview 1 introduced enhancements to the JIT compiler's devirtualization abilities for array interface methods, enabling the JIT to begin removing the abstraction overhead of array iteration via enumerators. Consider the following benchmarks:
```

Comment:
Should this say Preview 2? Or is this a Preview 1 feature we didn't call out before, or something being improved on furth in preview 2? This is confusing 😕

Reply:
> or something being improved on further in preview 2

It's this case. I'll add something here to transition into Preview 2-specific work.